[Ready for Publish] 4/20 Autocode (#37)
Contenda-Team/Contenda-landing-page: src/posts/autocode.md

---
layout: "../layouts/BlogPost.astro"

> title: "Why we love Autocode"

Reviewer: I think the title should be more "announcement-y," maybe like "Contenda + Autocode something something"

slug: autocode

> description: ""

Reviewer: ```suggestion
description: "Contenda is officially integrated into Autocode so you never have to worry about webhooks again!"
```

...or something

author: "Lilly Chen"
added: "April 12 2023"
tags: ["technical", "announcements"]
---

> ![Installing Contenda on Autocode](/blogimages/Autocode.gif)

Reviewer: I think the image here could go below the "why autocode" paragraph

# Why Autocode?

> Autocode seamlessly handles webhooks without any additional configuration. Due to compute-heavy AI features, Contenda uses webhooks to let you know when your job is finished. We wanted Contenda's programmatic features to be accessible to anyone, not just webhook-saavy users. By integrating with Autocode, you get a user-friendly IDE, template creation, key management, and great community support. 

Reviewer: ```suggestion
Autocode seamlessly handles webhooks without any additional configuration. Contenda has compute-heavy AI features, so when you submit a job to our service, we use webhooks to let you know when that job is finished.

We wanted Contenda's programmatic features to be accessible to anyone, not just webhook-saavy users, and thus, our official integration with Autocode was born!

If you use Contenda with our new integration with Autocode, you get a user-friendly IDE, template creation, key management, and great community support, right out of the box.
```

Reviewer: I just broke this up a little bit

## How do I get started?

> It's super simple! Just make an Autocode account and head over to the Contenda Autocode page. 

Reviewer: Add link to the page

> If you want to batch upload a YouTube playlist to Contenda, try this [template]()!

Reviewer: add link

> If you want to try making your own [Confbrew](https://contenda.co/blog/confbrew/), try this [template]()!

Reviewer: ```suggestion
In case you missed our most recent announcement, we launched [Confbrew](https://contenda.co/blog/confbrew/) recently, which allows you to take a playlist of conference videos, and create a Q&A bot from the content! If you'd like to build your own version, try this [template]()!
```

> If you have a plan and don't need no man, I'll take you straight to the [library docs]()!

Reviewer: add link

---
layout: "../layouts/BlogPost.astro"

> title: "Announcing Contenda + Autocode"

Reviewer: ```suggestion
title: "Announcing the Contenda + Autocode integration"
```

---
layout: "../layouts/BlogPost.astro"
title: "Announcing the Contenda + Autocode integration"
slug: autocode
description: "Contenda is officially integrated into Autocode so you never have to worry about webhooks again!"
author: "Lilly Chen"

> added: "April 12 2023"

Reviewer: ```suggestion
added: "April 20 2023"
```
